tests/Integration/Behaviour/Features/Scenario/Discount/BO/add_discount_with_brand_trigger.feature

```diff
@@ -0,0 +1,42 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s discount --tags add-discount-with-brand-trigger
+@restore-all-tables-before-feature
+@restore-languages-after-feature
+@add-discount-with-brand-trigger
+Feature: Add discount with brand trigger
+  PrestaShop allows BO users to update discounts conditions
+  As a BO user
+  I must be able to create discounts
+
+  Background:
+    Given shop "shop1" with name "test_shop" exists
+    Given there is a currency named "usd" with iso code "USD" and exchange rate of 0.92
+
+  Scenario: Create discount with brand trigger
+    Given I add product "beer_product" with following information:
+      | name[en-US] | bottle of beer |
+      | type        | standard       |
+    And I add product "potato_chips_product" with following information:
+      | name[en-US] | potato chips |
+      | type        | standard     |
+    And I add new manufacturer "rocket" with following properties:
+      | name                     | Rocket                             |
+      | short_description[en-US] | Cigarettes manufacturer            |
+      | description[en-US]       | Morbi at nulla id mi gravida blandit a non erat. Mauris nec lorem vel odio sagittis ornare.|
+      | meta_title[en-US]        | You smoke - you die!               |
+      | meta_description[en-US]  | The sun is shining and the weather is sweet|
+      | enabled                  | true                               |
+    When I create a "free_shipping" discount "discount_with_brand_trigger" with following properties:
+      | name[en-US] | Promotion |
+    Then discount "discount_with_brand_trigger" should have the following properties:
+      | name[en-US] | Promotion     |
+      | type        | free_shipping |
+    When I update discount "discount_with_brand_trigger" with following conditions matching at least 42 products:
+      | condition_type | items  |
+      | manufacturers  | rocket |
+    Then discount "discount_with_brand_trigger" should have the following properties:
+      | name[en-US]              | Promotion     |
+      | type                     | free_shipping |
+      | minimum_product_quantity | 0             |
+    Then discount "discount_with_brand_trigger" should have the following product conditions matching at least 42 products:
+      | condition_type | items      |
+      | manufacturers  | rocket |
```

Comment:
Ok for the BO part (except the product creation that seems useless)
Can you add a scenario in the FO sub part to check that the condition works as expected (create a product associated to a brand, create a discount based on a brand, check that adding the product triggers the discount, ...)